examples/text-generation/run_generation.py

```diff
@@ -163,7 +178,7 @@ def main():
 
     set_seed(args.seed)
 
-    tokenizer = AutoTokenizer.from_pretrained(args.model_name_or_path)
+    tokenizer = AutoTokenizer.from_pretrained(args.model_name_or_path, add_prefix_space=True)
```

Comment:
```suggestion
    if args.bad_words_ids is not None or args.force_words_ids is not None:
        tokenizer = AutoTokenizer.from_pretrained(args.model_name_or_path, add_prefix_space=True)
    else:
        tokenizer = AutoTokenizer.from_pretrained(args.model_name_or_path)
```
Let's add a prefix space only if needed for bad /forced words.